src/boot/ws.js

```diff
@@ -128,13 +128,37 @@ onWebSocket(dbEnum.wsCategory.sessionCreationError, (data) => {
   </center>`
   Notify.create({
     message: html,
-    color: 'negative',
+    color: 'black',
     position: 'top',
     html: true,
     timeout: 0,
   })
 
-  console.log(`sessionCreationError: ${JSON.stringify(data)}`)
+  console.log(`upgrade ${JSON.stringify(data)}`)
+})
+
+//commented unnecessary logs and listeners
+
+// onWebSocket(dbEnum.wsCategory.validation, (data) => {
+//   // console.log(`Validation recieved: ${data}`)
+// })
+
+onWebSocket(dbEnum.wsCategory.upgrade, (data) => {
+  let html = `<center>
+  <strong>${data}</strong>
+  <br>
+  'upgrade'
+  </center>`
+  Notify.create({
```

Comment:
Let's add a Cypress unit test for coverage.

Reply:
Good idea, on it